coreferee/lang/fr/language_specific_rules.py

```diff
@@ -797,8 +797,8 @@ def is_reflexive_anaphor(self, token: Token) -> int:
                     det
                     for det in token.children
                     if det.pos_ == "DET"
-                    and self.has_morph(token, "Poss", "Yes")
-                    and self.has_morph(token, "Person", "3")
+                    and self.has_morph(det, "Poss", "Yes")
+                    and self.has_morph(det, "Person", "3")
                 ]
```

Comment:
This definitely makes sense, although there doesn't seem to be a test that reacts to the change in any way.